docs/user-guide/ingest-data/for-observability/opentelemetry-collector.md

```diff
@@ -0,0 +1,87 @@
+---
+keywords: [OpenTelemetry Collector, OpenTelemetry, traces, data pipeline]
+description: Introduction to using OpenTelemetry Collector to write traces data to GreptimeDB.
+---
+
+# OpenTelemetry Collector
+
+## Overview
+
+[OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) is a vendor-neutral service for collecting and processing OpenTelemetry data. In this article, we will introduce how to use OpenTelemetry Collector to write traces data to GreptimeDB.
+
+## Quick Start
+
```

Comment:
Before starting the OpenTelemetry Collector, please ensure that GreptimeDB is running.